src/System.Private.DataContractSerialization/src/System/Xml/ValueHandle.cs

```diff
@@ -475,11 +475,7 @@ public byte[] ToByteArray()
 
         public string GetString()
         {
-            ValueHandleType type = _type;
-            if (type == ValueHandleType.UTF8)
```

Comment:
Then should it be removed from the switch? It doesn't need to be in both places.